### PR TITLE
NEW Accountancy - Add the possibility to have multiple personalized reports - SQL Part

### PR DESCRIPTION
**Pull request by aspangaro**
Preparing the ground to manage multiple personalized reports in accountancy.

### CODE REVIEW

File: htdocs/install/mysql/data/llx_accounting_category.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
--- Copyright (C) 2016 		Alexandre Spangaro   <aspangaro@open-dsi.fr>
--- Copyright (C) 2017 		Laurent Destailleur  <eldy@users.sourceforge.net>
+-- Copyright (C) 2016-2024	Alexandre Spangaro			<alexandre@inovea-conseil.com>
```
Comment (eldy) [inline htdocs/install/mysql/data/llx_accounting_category.sql:1]: Can you rename the file llx_accounting_category into
llx_accounting_category-accounting.sql
So table will be created when module accounitngis enabled.
We must start to have table name suffixed with module name after the "-".

File: htdocs/install/mysql/tables/llx_accounting_category_link.key.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+-- ===================================================================
```
Comment (eldy) [inline htdocs/install/mysql/tables/llx_accounting_category_link.key.sql:1]: For table that are link tables, we used to use the name of both joined tables into the name of table link.
So can you name it 
llx_accounting_category_account-accounting.sql
so
llx_modulename_table1_table2-modulename.sql

File: htdocs/install/mysql/data/llx_accounting_category.sql (hunk 1 above)
Comment (aspangaro) [reply htdocs/install/mysql/data/llx_accounting_category.sql]: Done. Thanks @eldy

File: htdocs/install/mysql/tables/llx_accounting_category_link.key.sql (hunk 2 above)
Comment (aspangaro) [reply htdocs/install/mysql/tables/llx_accounting_category_link.key.sql]: Done. Thanks @eldy

**Pull request merged**